### ISSUE TITLE
Both of the views working as sync.. Does it work on everyone..

### ISSUE THREAD

**Issue by WJJamshiD** [hellopyplane/Intro-to-Djang-async-views#1]

**Comment by scheung38** [comment]
Yes also 7 secs regardless of localhost:8000/sync or localhost:8000/async when in Django 3.2. Why is that?

Django           3.2.3
daphne           3.0.2

python3 --version                  
Python 3.9.5

daphne async_proj.asgi:application

**Comment by deep0072** [comment]
both functions are running synchronously. i have not seen any change yet
